### PR TITLE
adding bluechip chain to registry

### PR DESCRIPTION
**Pull request by Bluechip23**
New push for bluechip with updates via recommendations from first push.

### CODE REVIEW

File: bluechip/chain.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          "id": "3x4pynlp86prhcmtns742kgsgu7pjtzjpk8d93",
+          "address": "http://67.28.8.88:26659/",
+          "provider": "CryptoDungeon"
+        }
+      ]
+    },
+    "images": [
+      {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluechip/images/bluechip.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluechip/images/bluechip.svg"
```
Comment (JeremyParish69) [inline bluechip/chain.json:71]: I think the svg might be invalid

File: bluechip/chain.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "bluechip",
+    "status": "live",
+    "website": "https://www.bluechip.link",
+    "network_type": "mainnet",
+    "pretty_name": "BlueChip",
+    "chain_id": "bluechip_1",
+    "bech32_prefix": "bcp",
```
Comment (JeremyParish69) [inline bluechip/chain.json:9]: Pending registration: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Comment (Bluechip23) [reply bluechip/chain.json:9]: I have made a pull request on the registry

Comment (JeremyParish69) [reply bluechip/chain.json:9]: I see it registered as "bluechip", not as "bcp". Please make sure they are both the same and match what the chain uses.

Comment (Bluechip23) [reply bluechip/chain.json:9]: I misread the column. I have updated my information on SLIP table.

File: bluechip/chain.json (hunk 1 above)
Comment (JeremyParish69) [reply bluechip/chain.json:71]: Please remove the svg. It looks low quality and unlike the png.

Comment (JeremyParish69) [reply bluechip/chain.json:71]: The svg is not scalable vectors, it's just an embedded image. Please remove unless there is an actual scalable vector file.

Comment (Bluechip23) [reply bluechip/chain.json:71]: I have deleted it. I will have to find a resource to help make an svg file. Is this something required for initial submittal?

Comment (JeremyParish69) [reply bluechip/chain.json:71]: I think the last thing is to remove iamges[0].svg from this chain file and from the assetlist.

File: bluechip/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            "exponent": 6
+          }
+        ],
+        "base": "ubluechip",
+        "name": "blue chip",
+        "display": "bcp",
+        "symbol": "BCP",
+        "images": [
+          {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluechip/images/bluechip.png",
```
Comment (JeremyParish69) [inline bluechip/assetlist.json:23]: this comma will need to be removed

**Pull request merged**